packages/data-context/src/util/migration.ts

```diff
@@ -78,3 +137,72 @@ function createTestingTypeTemplate (testingType: 'e2e' | 'component', pluginPath
     ${formatObjectForConfig(options, 4)}
   },`
 }
+
+export interface RelativeSpecWithTestingType {
+  testingType: TestingType
+  relative: string
+}
+
+async function findByTestingType (cwd: string, dir: string | null, testingType: TestingType) {
+  if (!dir) {
+    return []
+  }
+
+  return (await globby(`${dir}/**/*`, { onlyFiles: true, cwd }))
+  .map((relative) => ({ relative, testingType }))
+}
+
+export async function getSpecs (
+  projectRoot: string,
+  componentDirPath: string | null,
+  e2eDirPath: string | null,
+): Promise<{
+  before: RelativeSpecWithTestingType[]
+  after: RelativeSpecWithTestingType[]
+}> {
+  const [comp, e2e] = await Promise.all([
+    findByTestingType(projectRoot, componentDirPath, 'component'),
+    findByTestingType(projectRoot, e2eDirPath, 'e2e'),
+  ])
+
+  return {
+    before: [...comp, ...e2e],
+    after: [...comp, ...e2e].map((x) => {
+      return {
+        testingType: x.testingType,
+        relative: renameSpecPath(x.relative),
+      }
+    }),
+  }
+}
+
+export async function moveSpecFiles (e2eDirPath: string) {
+  const specs = (await getSpecFiles(e2eDirPath)).map((spec) => {
+    const specPath = `${e2eDirPath}/${spec}`
+
+    return {
+      from: specPath,
+      to: renameSpecPath(specPath),
+    }
+  })
+
+  specs.forEach((spec) => {
+    fs.moveSync(spec.from, spec.to)
+  })
+}
+
+async function getSpecFiles (dirPath: string) {
```

Comment:
FYI this is basically just re-inventing the `globby` library @ImCesar. Just do

```js
await globby('**/*/your-glob', { filesOnly: true })
```

It's free!

Reply:
After messing around with the e2e test I was wondering if this is the path we want to go. Do we only want files for this? If they have a nested directory like we have in our test project with `someDir/someFile` do we want to rename those as well? In the UI it shows them being renamed but we haven't renamed them